src/frontend/src/components/RecordSearch/Record/DispositionQuestion.tsx

```diff
@@ -16,28 +16,70 @@ interface State {
   conviction_date: string;
   probation_revoked_date: string;
   missingFields: boolean;
-  invalidDate: boolean
+  invalidDate: boolean;
+  submitPending: boolean;
 }
 
 class DispositionQuestion extends React.Component<Props, State> {
   state: State  = {
-    status: "Open",
+    status: "Unknown",
     conviction_date: "",
     probation_revoked_date: "",
     missingFields: false,
-    invalidDate: false
+    invalidDate: false,
+    submitPending: false
   };
 
   componentDidMount() {
     this.setState({
-      status: (this.props.disposition ? this.props.disposition.status : "Open") ,
+      status: (this.props.disposition ? this.props.disposition.status : "Unknown") ,
       conviction_date: (this.props.disposition ? this.props.disposition.date : ""),
       }
     )
   }
 
-  handleChange = (e: React.BaseSyntheticEvent) => {
+  handleDismissedClick = () => {
     this.setState<any>({
+      ["status"]: "Dismissed",
+      ["conviction_date"]: "",
+      ["probation_revoked_date"]: "",
+      ["missingFields"]: false,
+      ["invalidDate"]: false,
+      ["submitPending"]: false
+    }, this.dispatchAnswer);
+  };
+
+  handleConvictedClick = () => {
+    this.setState<any>({
+      ["status"]: "Convicted",
+      ["probation_revoked_date"]: "",
+      ["missingFields"]: false,
+      ["invalidDate"]: false,
+      ["submitPending"] : true
+    });
+  };
+
+  handleRevokedClick = () => {
+    this.setState<any>({
+      ["status"]: "revoked",
+      ["submitPending"] : true
+    });
+  };
+
+  handleUnknownClick = () => {
+    this.setState<any>({
+      ["status"]: "Unknown",
```

Comment:
I'm used to

```
this.setState({"key": "value", ...})
```

but what is the difference compared to

```
this.setState({["key"]: "value", ...})
```

Reply:
Ah, I was copying the format from line 83, `[e.target]: e.value` , in which a variable string key needs to be in brackets. But a string literal key doesn't, so I've removed those. Thanks.